apps/prairielearn/src/schemas/schemas/infoAssessment.json

```diff
@@ -163,9 +165,11 @@
     }
   },
   "definitions": {
-    "groupRole": {
+    "GroupRoleJsonSchema": {
       "description": "A custom role for use in group assessments that allows control over certain permissions.",
       "type": "object",
+      "additionalProperties": false,
+      "required": ["name"],
```

Comment:
Note to self, check if this will be breaking (I should really run all existing JSON files through these new schemas before we merge this).